homeassistant/components/saj/translations/en.json

```diff
@@ -0,0 +1,24 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```

Comment:
If I'm not mistaken the translations still need to be filled in this file?
The error I get when testing is literally the key and not a translated string...

Reply:
fixed